.github/workflows/autoupdate.yml

```diff
@@ -3,9 +3,12 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    types: [ labeled ]    
```

Comment:
It's redundant to use both `types` and `PR_FILTER` because they both serve the same purpose of filtering the pull requests that should trigger the workflow.